samples/Sentry.Samples.AspNetCore.Blazor.Wasm/Program.cs

```diff
@@ -28,6 +28,6 @@
 catch (Exception e)
 {
     SentrySdk.CaptureException(e);
-    await SentrySdk.FlushAsync(TimeSpan.FromSeconds(2));
+    await SentrySdk.FlushAsync();
```

Comment:
Since we're showcasing it here, does it make sense to add setting the timeout to the `Init`?

Reply:
The flush is part being showcased, not the timeout.  In general, the point with most of this PR is that the developer either should set the timeout once in options and not think about it again, or have a very specific reason to deviate from the default.